src/main/java/org/fundacionjala/coding/nestor/FizzBuzz.java

```diff
@@ -0,0 +1,45 @@
+package org.fundacionjala.coding.nestor;
+
+import java.util.Arrays;
+
+/**
+ * Class Fizz Buzz.
+ */
+public class FizzBuzz {
+  public static final int F = 5;
+  public static final int T = 3;
+  /**
+   * Method to transform numbers % 3 and numbers % 5.
+   *
+   * @param counter type Array of Integer.
+   * @return type Array of String.
+   */
+  public String[] fizzBuzz(final Integer[] counter) {
+    return Arrays.stream(counter)
+        .map(n -> isDivBy(F, T, n) ? "FizzBuzz" : isDivBy(F, n) ? "Buzz" : isDivBy(T, n) ? "Fizz" : String.valueOf(n))
```

Comment:
![MAJOR](https://sonarsource.github.io/sonar-github/severity-major.png 'Severity: MAJOR') Extract this nested ternary operation into an independent statement. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=squid%3AS3358)
![MAJOR](https://sonarsource.github.io/sonar-github/severity-major.png 'Severity: MAJOR') Extract this nested ternary operation into an independent statement. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=squid%3AS3358)